### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -29,46 +29,46 @@ jobs:
...
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
       - name: Install Node.js ${{ matrix.node }}
         uses: actions/setup-node@v1
         with:
-          node-version: ${{ matrix.node }}
+          node: ${{ matrix.node }}
```
Comment (mysticatea) [inline .github/workflows/ci.yml:71]: The original `node-version` is the correct parameter name.

Comment (MichaelDeBoey) [reply .github/workflows/ci.yml:71]: The checkout step just uses `node`, that's why I changed it here too

Comment (mysticatea) [reply .github/workflows/ci.yml:71]: For example, https://github.com/mysticatea/eslint-plugin-es/runs/666749068?check_suite_focus=true#step:3:5, we can see the `actions/setup-node` ignores the `node` parameter.

**Comment by mysticatea** [comment]
By the way, I like appending `.x` for version numbers. The default value of `actions/setup-node` is so, too.

**Pull request merged**